content/raw/general/onboarding.md

```diff
@@ -0,0 +1,260 @@
+- The following options MUST be displayed in the login screen
+    - Add new user
+    - Add existing Status user
+    - Lost Keycard
+- If and only if the type of login of the selected user is password then the "Password Login" screen MUST BE displayed
+- If and only if the type of login of the selected user is Touch ID then the "Biometrics Login" screen MUST BE displayed
+- If and only if the type of login of the selected user is Keycard then the "Keycard Login" screen MUST BE displayed
+
+#### Successful login screen
+
+- If the user logins sucessuflly then a loading screen MUST BE displayed
+
+#### Password Login screen
+
+- Then the password field MUST be displayed
+- If the user types a wrong password the error "Password incorrect" MUST BE displayed
+
+####  Biometrics Login screen
+
+- The biometrics logo MUST BE shown only if the platform is MacOS
+- The message "Waiting for TouchID." MUST BE displayed
+- The option for "Use password instead" MUST BE displayed
+- If "Use password instead" is selected the password field MUST BE displayed, this should behave like the password login screen
+- If "Use password instead" is selected the Touch ID logo MUST BE displayed, clicking this logo MUST take the user back to the original screen "Waiting for Touch ID"
+- If the user biometric login is NOT sucesfull then the biometrics logo MUST BE replaced with a logo indicating an error
+- If the user biometric login is NOT sucesfull then a error message "Fingerprint not recognized" MUST BE displayed
+- If the user biometric login is NOT sucesfull then the option "Use password instead" MUST BE displayed
+
+#### Keycard Login screen
+
+- The keycard logo MUST BE shown
+- If no reader is detected the message "Plug in Keycard reader.." MUST be displayed
+- If a reader is detected but no keycard is present the message "Insert your Keycard" MUST be displayed
+- If a keycard is present then the loading message "Reading Keycard..." MUST be displayed
+- If the keycard is not linked to the seleted account then the error message "Wrong Keycard! The card inserted is not linked to your profile" MUST BE displayed
+- If the keycard is not linked to the seleted account then the message "Insert another Keycard" MUST BE displayed
+- If the keycard is empty then the error message "The card inserted is empty" MUST BE displayed
+- If the keycard is empty then the option "Generate keys for a new Keycard" MUST BE displayed
+    - TODO
+- If the keycard is locked then the error message "Keycard locked" MUST BE displayed
+    - TODO
+- If the keycard is locked then the option "Unlock your Keycard" MUST BE displayed
+    - TODO
+- If the keycard is locked with a PIN then a "Enter Keycard PIN" message MUST BE displayed
+- If the keycard is locked with a PIN then a 6 Digit input MUST BE displayed
+- If the keycard pin is incorrect then the error message "PIN incorrect" MUST be displayed
+- The maximum number of possible PIN attempts MUST BE 3
```

Comment:
@iurimatias I am not 100% sure, but new keycard should be limited to 5 tries instead of 3 (anyhow that's the number coming from the keycard, not something that we have hardcoded, so whatever the card says we're displaying that), but just wanted to mention that because of this requirement.